014-realtime-api.md

```diff
@@ -112,24 +172,30 @@ On each new connection, we should validate that the client origin is valid simil
 
 #### Limit payload size
 
+Since we will not support 2-way communication now - we shouldn't allow/handle receiving messages at all for now. Which makes the payload size not a concern for now.
```

Comment:
Maybe we should response to incoming messages with a "warning" or at least a note?

Reply:
I would prefer the very strict way and just disconnect the client. This leaves no room for abuse.